spinalcordtoolbox/reports/qc.py

```diff
@@ -641,12 +660,19 @@ def generate_qc(fname_in1, fname_in2=None, fname_seg=None, angle_line=None, args
         qcslice_operations = [QcImage.template]
         qcslice_layout = lambda x: x.mosaic()
     # Sagittal orientation, display vertebral labels
-    elif process in ['sct_label_vertebrae']:
+    elif process in ['sct_label_vertebrae',]:
         plane = 'Sagittal'
         dpi = 100  # bigger picture is needed for this special case, hence reduce dpi
         qcslice_type = qcslice.Sagittal([Image(fname_in1), Image(fname_seg)], p_resample=None)
         qcslice_operations = [QcImage.label_vertebrae]
         qcslice_layout = lambda x: x.single()
+    #  Sagittal orientation, display posterior labels
+    elif process in ['sct_label_utils']:
+        plane = 'Sagittal'
+        dpi = 100  # bigger picture is needed for this special case, hence reduce dpi
+        qcslice_type = qcslice.Sagittal([Image(fname_in1), Image(fname_seg)], p_resample=None)
```

Comment:
We could perform the projection here (into the relevant slice) And transforms this. This could look like:
~~~
elif process in ['sct_label_utils']:
        plane = 'Sagittal'
        dpi = 100  # bigger picture is needed for this special case, hence reduce dpi
        projected_image = projection (Image(fname_seg))
        qcslice_type = qcslice.Sagittal([Image(fname_in1), projected_image], p_resample=None)
        qcslice_operations = [QcImage.label_utils]
        qcslice_layout = lambda x: x.single()